packages/hooks/src/useHasSwapFreeAccount.ts

```diff
@@ -0,0 +1,15 @@
+import { useStore } from '@deriv/stores';
+
+// This hook is used to check if the client has a Swap-Free account
+// It checks for availability of market_type 'all' in trading_platform_available_accounts API response
+const useHasSwapFreeAccount = () => {
+    const { client } = useStore();
+    const { trading_platform_available_accounts } = client;
+    const has_swapfree_account = trading_platform_available_accounts.some(
```

Comment:
```suggestion
    const has_swap_free_account = trading_platform_available_accounts.some(
```
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
sure